ember-cli-build.js

```diff
@@ -5,6 +5,10 @@ var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
     // Add options here
+    babel: {
+      includePolyfill: true
```

Comment:
What'd we do that resulted in needing the polyfill? This is something I don't want us to have to ship.